profiler/shakesapp/README.md

```diff
@@ -0,0 +1,32 @@
+# Profiler Shakespeare Application in Go
+
+This application starts a server which will return the number of times a word or
+phrase appears in the works of Shakespeare, and sends queries to that server to
+create load.
+
+Profiler is enabled for this application, and can be used to identify
+opportunities to optimize the server code.
+
+## Running the application
+
+This application should be run with Go 1.14. Once that version of Go is
```

Comment:
This will obsolete when Go 1.15 is out. Do we care to be that prescriptive?

Reply:
Removed.